app/routes/events/view/edit.js

```diff
@@ -21,7 +21,8 @@ export default Route.extend(AuthenticatedRouteMixin, EventWizardMixin, {
       types : await this.store.query('event-type', {
         sort: 'name'
       }),
-      topics: await this.store.query('event-topic', {
+      module : await this.get('store').queryRecord('module', {}),
```

Comment:
Use ES5 getters whenever possible. `this.store` instead of `this.get('store')`

Reply:
Sure